Controller/PageAdminController.php

```diff
@@ -94,7 +94,9 @@ public function treeAction()
 
         $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
 
-        return $this->render('SonataPageBundle:PageAdmin:tree.html.twig', array(
+        $templates = $this->retrieveAdminTemplates();
+
+        return $this->render($templates['tree'], array(
```

Comment:
What is the relation with PHP 5.3?

The line `$templates = $this->retrieveAdminTemplates();` is not needed anymore if you use the following code:

```php
return $this->render($this->admin->getTemplate('tree'), array());
```